docs/reference/feel/language-guide/feel-context-expressions.md

```diff
@@ -0,0 +1,116 @@
+---
+id: feel-context-expressions 
+title: Context expressions
+description: "This document outlines context expressions and examples."
+---
+
+### Literal
+
+Creates a new context with the given entries. Each entry has a key and a value. The key is either a
+name or a string. The value can be any type.
+
+```js
+{
+  a: 1,
+  b: 2
+}
+// {a:1, b:2}
+
+{
+  "a": 1,
+  "b": 2
+}
+// {a:1, b:2}
+```
+
+Inside the context, the previous entries can be accessed.
+
+```js
+{
+  a: 2,
+  b: a * 2
+}
+// {a:2, b:4}
+```
+
+A context value can embed other context values.
+
+```js
+{
+  a: 1,
+  b: {
+    c: 2
+  }
+}
+// {a:1, b:{c:2}}
+```
+
+### Get entry/path
```

Comment:
This title with a special character `/` does not create an anchor link like you would expect. I would recommend removing the special character like 

"Get entry or path" -> get-entry-or-path

Or 

"Get entry/path" should resolve to -> get-entrypath 

because `/` is simply ignored in markdown, but may behave differently in Docusaurus. 

If you choose the second option, watch the build process when you commit to GitHub to make sure it passes.

Reply:
^ This is also the source of your build error on the last commit @christinaausley

Reply:
Fixed this in the latest commit!